### DISCUSSION TITLE
Custom item action to list view

### DISCUSSION THREAD

**Discussion by GandonNicolas** [sulu/sulu#6816] [Q&A] [answered]
I am trying to create a custom action item, which I want to add to each element of a user list, which will allow me to send them a predefined email with a single click.

I tried to do it following the example given here : 
- https://github.com/sulu/sulu-demo/pull/83

I get an error when building the administration frontend application with the command :`bin/console sulu:admin:update-build`

`Error: PostCSS plugin postcss-hexrgba requires PostCSS 8.`
I really need to change this version in my package.json?

My code is identical to the one in the example. I'm just trying to reproduce it. I'll change the javascript actions afterwards.

Maybe I don't even need to create custom javascript for this feature (which would be fine ^^) 

Best regards
Nico

**Comment by luca-rath** [comment, marked as answer]
Hi Nico,
which Sulu version are you using? There are some older versions (which we do not support anymore) that had similar issues, if you're lower than 2.4, consider upgrading to 2.5 or at least to 2.4 ... If that's not the problem, make sure your `assets/admin` directory matches the `assets/admin` of the Sulu Skeleton (in your specific version), so that just your `assets/admin/app.js` has different contents compared to the one from sulu.

**Reply by GandonNicolas** [reply]
Hi Lucas,

I am using the `sulu/sulu" version: "~2.4.2"`

The structure of my `assets/admin` folder contains an extra folder with a file in it, like in the demo I followed: 
- https://github.com/sulu/sulu-demo/pull/83

So I tried to put all my code in the `app.js` file, delete the extra folder and file to respect the remote repo structure, then build but it still doesn't work.

**Reply by luca-rath** [reply]
Okay there was indeed a bug (also in 2.4) which has been fixed in 2.4.4 (see https://github.com/sulu/skeleton/pull/179) ... I tried it real quick and it's working with 2.4.5

**Reply by luca-rath** [reply]
And it's okay to have extra dirs and files in your `assets/admin`, just the files like `package.json` and `webpack.config.js` and all the others from the skeleton should in most cases match the ones in the skeleton.

**Reply by GandonNicolas** [reply]
I have upgraded to version 2.4.4 and it works fine. 

Thanks a lot for your help Lucas 🙏
